packages/uniforms-mantine/src/ListItemField.tsx

```diff
@@ -1,20 +1,24 @@
+import { List, Group, Box } from '@mantine/core';
 import React, { ReactNode } from 'react';
 import { connectField } from 'uniforms';
-import { List, Group, Box } from '@mantine/core';
 
 import AutoField from './AutoField';
 import ListDelField from './ListDelField';
 
 export type ListItemFieldProps = { children?: ReactNode; value?: unknown };
 
 function ListItem({
-  children = <AutoField label={null} name="" />,
+  children = <AutoField label={null} name="" style={{ width: '100%' }} />,
 }: ListItemFieldProps) {
   return (
-    <List.Item mb={12}>
-      <Group mt="xs" align="center">
+    <List.Item
+      mb={12}
+      styles={{ itemLabel: { width: '100%' }, itemWrapper: { width: '100%' } }}
+      pos="relative"
+    >
+      <Group mt="xs" align="flex-start" w="100%">
         {children}
-        <Box mt={20}>
+        <Box pos="absolute" right={0} top={10}>
```

Comment:
Can those styles be moved to the `ListDelField` component? We can avoid additional nesting with `Box`.